src/nativescript/user.js

```diff
@@ -1,8 +1,10 @@
 import { User as CoreUser } from '../core/user';
 import { getDataFromPackageJson } from './utils';
+import * as app from "application";
 
 export class User extends CoreUser {
   loginWithMIC(redirectUri, authorizationGrant, options) {
+    redirectUri = isInsidePreviewApp() ? "nsplayresume://" : redirectUri;
```

Comment:
Is the `nsplayresume://` scheme being setup as a custom url scheme for the app on iOS and an `IntentFilter` on Android?

Reply:
See https://github.com/Kinvey/js-sdk/blob/aab4721338b22a6c51d29b4246ea0fcb4d4d18aa/packages/kinvey-nativescript-sdk/platforms/ios/Info.default.plist#L12 for iOS and https://github.com/Kinvey/js-sdk/blob/aab4721338b22a6c51d29b4246ea0fcb4d4d18aa/packages/kinvey-nativescript-sdk/platforms/android/AndroidManifest.default.xml#L8 for Android

Reply:
Are we able to call `isInsidePreviewApp()` during a hook instead of in the code?

Reply:
Ok great!

I think my unfamiliarity with how the Preview App works is why I was asking about calling `isInsidePreviewApp()` in a hook.

I need to spend some time (today) playing with the preview app but what you have changed looks good to me.

I will approve it once I can play with the preview app with your changes.